lib/rpmsg/rpmsg.c

```diff
@@ -132,7 +132,7 @@ int rpmsg_send_ns_message(struct rpmsg_endpoint *ept, unsigned long flags)
 	ns_msg.flags = flags;
 	ns_msg.addr = ept->addr;
 	strncpy(ns_msg.name, ept->name, sizeof(ns_msg.name));
-	ret = rpmsg_send_offchannel_raw(ept, ept->addr,
+	ret = rpmsg_send_offchannel_raw(ept, RPMSG_NS_EPT_ADDR,
```

Comment:
Any rational to do this?
Looking at Linux implementation the source address seems the endpoint address not the NS endpoint address

Reply:
Yes, you are right. I will drop this patch.